src/app/technical-record/dimensions/dimensions.component.html

```diff
@@ -1,102 +1,128 @@
 <table class="govuk-table table-details">
   <thead class="govuk-table__head">
-  <tr class="govuk-table__row">
-    <th class="govuk-table__cell govuk-!-font-weight-bold">Length (mm)</th>
-    <td id="test-length" class="govuk-table__cell">
-      {{ activeRecord.dimensions.length | DefaultNullOrEmpty }}
-    </td>
-    <th class="govuk-table__cell"></th>
-    <th class="govuk-table__cell"></th>
-    <th class="govuk-table__cell"></th>
-    <th class="govuk-table__cell"></th>
-  </tr>
+    <tr class="govuk-table__row">
+      <th class="govuk-table__cell govuk-!-font-weight-bold">Length (mm)</th>
+      <td id="test-length" class="govuk-table__cell">
+        {{ activeRecord?.dimensions.length | DefaultNullOrEmpty }}
+      </td>
+      <th class="govuk-table__cell"></th>
```

Comment:
Alot of <th class="govuk-table__cell"></th> in this file. I think we should remove all if not used

Reply:
The complete component is designed as table. It needs the th to main the format